src/components/input/help-text.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import './help-text.scss';
+
+function HelpText(props) {
+  const { hasSuccess, hasWarning, hasError } = props;
+  const classes = classNames('help-text', {
+    'help-text--has-success': hasSuccess,
+    'help-text--has-warning': hasWarning,
+    'help-text--has-error': hasError,
+  });
+
+  if (props.children) {
+    return <span className={ classes }>{ props.children }</span>;
+  }
+
+  return <span />; // Stateless functions do not support returning null, see https://github.com/facebook/react/issues/5355
```

Comment:
I know, should probably change the line comment to `Stateless functions in React 0.14.X do not support returning null, see https://github.com/facebook/react/issues/5355`